<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommunicationWidget</class>
 <widget class="QWidget" name="CommunicationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>174</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>330</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CommunicationWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#CommunicationWidget {
	background-color: transparent;
}

QWidget#chatControlsWidget {
	background-color: rgba(0,0,0,175);
	border-top: 1px solid rgba(255,255,255,200);
	border-right: 1px solid rgba(255,255,255,200);
	border-top-right-radius: 10px;
}

QWidget#chatContentWidget {
	background-color: transparent;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="mainVerticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="chatContentWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="contentContainerLayout">
          <property name="margin">
           <number>6</number>
          </property>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="chatControlsWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="margin">
         <number>6</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="chatHorizontalLayout">
          <item>
           <widget class="QLineEdit" name="chatLineEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="viewModeButton">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>History</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="imButton">
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>IM</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
